openrouteservice/src/main/java/org/heigit/ors/routing/RoutingProfileManager.java

```diff
@@ -625,12 +625,12 @@ public MatrixResult computeMatrix(MatrixRequest req) throws Exception {
     }
 
     public void createRunFile() {
-        File file=new File("ors.run");
+        File file = new File("ors.run");
         try (FileWriter fw = new FileWriter(file)) {
             fw.write("ORS init complete: "+ Instant.now().toString() + "\n");
             fw.flush();
-        } catch(IOException ex) {
-            ex.printStackTrace();
+        } catch(Exception ex) {
```

Comment:
Isn't it better to catch an explicit exception rather than the generic one (Sonar lint always brought that up for me anyway)?

Reply:
normally yes, I don't see the reasoning since it's just a "ignore any errors" and it would require several different catch blocks for the same thing which is simply annoying...